SPECS/gh/gh.spec

```diff
@@ -32,6 +32,7 @@ Patch0:         fix-relative-time-search-tests.patch
 Patch1:         CVE-2021-43565.patch
 Patch2:         CVE-2022-32149.patch
 Patch3:         CVE-2024-54132.patch
+Patch4:         CVE-2024-45338.patch
```

Comment:
Unfortunately, there is no autosetup/autopatch in this spec, you'll need to apply it manually or even better: replace existing patch commands with `%autopatch -p1`

Reply:
Thanks, will fix this.

Reply:
thanks rohit for catching this. sumynwa, can you add `%autopatch -p1` instead of just adding entry for your patch below

Reply:
Fixed.